src/ui/locales/uk.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Показувати атрибуцію",
```

Comment:
```suggestion
    "AttributionControl.ToggleAttribution": "Показати атрибуцію",
```
 based on https://mapbox.slack.com/archives/C02G2QA3N/p1649760507147809?thread_ts=1649701336.272749&cid=C02G2QA3N